roles/telemetry_graphing/tasks/cypress_test.yml

```diff
@@ -0,0 +1,87 @@
+- name: Install cypress and the dependencies
+  block:
+    - name: Install Node.js
+      ansible.builtin.dnf:
+        name: nodejs
+        state: present
+      become: true
+
+    - name: Install dependencies
+      ansible.builtin.dnf:
+        name:
+          - xorg-x11-server-Xvfb
+          - gtk3-devel
+          - nss
+          - alsa-lib
+        state: present
+      become: true
+
```

Comment:
This is listed as a pre-req in the README. You should list it in the role description as something the role does, or remove it from here, and add it into the playbook in ci/

Reply:
I will update README file